octobot_tentacles_manager/api/util/tentacles_management.py

```diff
@@ -19,14 +19,17 @@
 import octobot_tentacles_manager.util as util
 
 
-async def manage_tentacles(worker, tentacle_names, tentacles_path_or_url=None, aiohttp_session=None) -> int:
+async def manage_tentacles(worker, tentacle_names, tentacles_path_or_url=None, aiohttp_session=None,
+                           authenticator=None) -> int:
     errors_count = 0
     logger = logging.get_logger(__name__)
     try:
         if tentacles_path_or_url is not None:
+            session = authenticator.get_aiohttp_session() if authenticator and authenticator.is_logged_in() \
```

Comment:
:+1:

Reply:
Where is this Authenticator class defined ? In Commons ?

Reply:
here https://github.com/Drakkar-Software/OctoBot/blob/master/octobot/community/authentication.py

Reply:
Ok

Reply:
I don't like to manipulate object defined in the caller. I would expect to have this Authenticator defined in Commons. But if it's only for this call it's probably overkill to move this class declaration.

Reply:
It's strictly related to the octobot community features so to me it makes sense to have it in octobot, we could define an abstract class in octobot-commons though

Reply:
:+1: